### PR TITLE
Itemlink tutorial improvement

### PR DESCRIPTION
**Pull request by Joethepic**

### CODE REVIEW

**Comment by Berserker66** [comment]
@alwaysintreble

**Comment by alwaysintreble** [comment]
Can you put this in the advanced guide rather than the plando guide? Item links aren't plando so it belongs there. There's an existing example already but another example would be helpful along with a proper description in the section describing settings that can be used across all games.

**Comment by Joethepic** [comment]
I was pretty sure it was already in the advanced yaml guide it even matches what the advanced yaml guide is if it isn’t which file is the advanced yaml guide

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [inline worlds/generic/docs/advanced_settings_en.md:128]:
```suggestion
* `item_links` allows players to link their items into a group with the same item link name and game. The items declared in `item_pool` get combined and when an item is found for the group, all players in the group receive it. Item links can also have local and non local items, forcing the items to either be placed within the worlds of the group or in worlds outside the group. If players have a varying amount of a specific item in the link, the lowest amount from the players will be the amount put into the group.
```

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [inline worlds/generic/docs/advanced_settings_en.md:150]:
```suggestion
  A Link to the Past: 10
```

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [inline worlds/generic/docs/advanced_settings_en.md:153]:
```suggestion
  version: 0.3.2
```
since this is using item links this also needs to be updated for consistency.

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [inline worlds/generic/docs/advanced_settings_en.md:194]: Lines 195-281 should be removed, since they don't add any value.

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [inline worlds/generic/docs/advanced_settings_en.md:340]: The Link to the Past and Timespinner item_links are two separate sections and should be documented as such.

**Review by alwaysintreble — CHANGES_REQUESTED** [review]
Some changes I'd like to see to help with consistency and ease of access.

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [reply worlds/generic/docs/advanced_settings_en.md:253]: this was not fixed correctly.

Comment (Joethepic) [reply worlds/generic/docs/advanced_settings_en.md:253]: I think I fixed it now. Let me know if there is any other way to improvements that you suggest.

**Comment by Ijwu** [comment]
Waiting for @alwaysintreble's approval on this one, I'm just going to skip on by for now. I'll skim this once they've approved and then merge in if it looks good.

**Comment by alwaysintreble** [comment]
👍

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:241]: Missing space between "the" and "`Twin Pyramid Key`"

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:243]: > The `null` replacement item instead of forcing a filler item will allow the generator to fill the item.

The last sentence here doesn't make sense to me. Please elaborate or re-write.

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:210]: The sentence says they have an equal chance, then that they each have 10/20 chance. Instead of restating chances, please describe in one clause that they are equal 10/20 chances.

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [reply worlds/generic/docs/advanced_settings_en.md:128]: You did nothing with this feedback. Either address it directly or take the suggestion. Resolving a conversation without a word is discouraged.

**Review by Ijwu — CHANGES_REQUESTED** [review]

File: worlds/generic/docs/advanced_settings_en.md
Comment (Joethepic) [reply worlds/generic/docs/advanced_settings_en.md:243]: I worded it a bit different to fix it

File: worlds/generic/docs/advanced_settings_en.md
Comment (alwaysintreble) [reply worlds/generic/docs/advanced_settings_en.md]: He actually did sort of write what i suggested but differently then I ended up PRing it in with different wording and you've critiqued some of my writing now 😄

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:211]:
```suggestion
* `requires` is set to required release version 0.3.2 or higher.
```

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:245]:
```suggestion
Key` and `Timespinner Wheel` will be forced among the worlds of those in the group. The `null` replacement item will, instead
```

File: worlds/generic/docs/advanced_settings_en.md
Comment (Ijwu) [inline worlds/generic/docs/advanced_settings_en.md:246]:
```suggestion
of forcing a specific chosen item, allow the generator to randomly pick a filler item in place of putting in another one of the linked item.
```

**Review by Ijwu — CHANGES_REQUESTED** [review]

**Comment by Joethepic** [comment]
should of be fixed now used the commit button on all those suggestions

**Review by Ijwu — APPROVED** [review]

**Pull request merged**
